### PR TITLE
2541 adds whats new 0.6

### LINKED ISSUE #2541: adds a 'what's new in 0.6.0' page
**Is your feature request related to a problem? Please describe.**
part of #2477, based on #2490

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

Fixes #2541 

### Status
**ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Comment by wyli** [comment]
Hi @Nic-Ma @yiheng-wang-nv, I'll tag 0.6 after merging this one. do you have any other pending PRs?

File: docs/source/whatsnew_0_6.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## Pythonic APIs to load the pretrained models from Clara Train MMARs
+[The MMAR (Medical Model ARchive)](https://docs.nvidia.com/clara/clara-train-sdk/pt/mmar.html)
+defines a data structure for organizing all artifacts produced during the model development life cycle.
+NVIDIA Clara provides [various MMARs of medical domain-specific models](https://ngc.nvidia.com/catalog/models?orderBy=scoreDESC&pageNumber=0&query=clara_pt&quickFilter=&filters=).
+These MMARs include all the information about the model including configurations and scripts to provide a workspace to perform model development tasks. To better leverage the trained MMARs released on Nvidia GPU cloud, MONAI provides pythonic APIs to access them.
+
+To demonstrate this new feature, a medical image segmentation tutorial is created within
+[`project-monai/tutorials`](https://github.com/Project-MONAI/tutorials/blob/master/modules/transfer_mmar.ipynb)).
+It mainly produces the following figure to compare the loss curves and validation scores for
+(1) training from scratch (the green line),
```
Comment (Nic-Ma) [inline docs/source/whatsnew_0_6.md:33]: Could you please help unify the sub index type? I see `1, 2, 3`, `(1), (2), (3)`, and `-, -, -` in this document.

Thanks.

**Review by Nic-Ma — COMMENTED** [review]
Looks good to me except for a minor comment.

Thanks.

**Review by Nic-Ma — APPROVED** [review]
Thanks for the quick update.

**Comment by Nic-Ma** [comment]
I think @yiheng-wang-nv and I don't have other pending PRs.
Please feel free to tag v0.6.

Thanks.

**Comment by wyli** [comment]
> I think @yiheng-wang-nv and I don't have other pending PRs.
> Please feel free to tag v0.6.
> 
> Thanks.

Hi @Nic-Ma, I reverted a980ae4d98ef2742a5c1352fff1d11de6640c227, because the integration test failed. #2531 is not a critical feature, so let's postpone it to after v0.6.0

**Comment by Nic-Ma** [comment]
Hi @wyli ,

Sure, it's not a blocking issue I think.

Thanks.

**Comment by wyli** [comment]
> Hi @wyli ,
> 
> Sure, it's not a blocking issue I think.
> 
> Thanks.

thanks for confirming, then I'll release 0.6 now

**Pull request merged**